hack/update-codegen.sh

```diff
@@ -201,6 +201,28 @@ gardenlet_groups() {
 }
 export -f gardenlet_groups
 
+# Componentconfig for landscaper-gardenlet
+
+landscaper_groups() {
```

Comment:
```suggestion
landscapergardenlet_groups() {
```
If there are more landscaper groups in the future, they should be in separate functions to increase the level of parallelism (the more generation that can happen in parallel, the better).